src/components/Extensions/api/internal/useExtensionsData.tsx

```diff
@@ -17,25 +17,54 @@ import { Extension } from '../../models';
 import { ExtensionBackend, isLocalExtension } from '../useExtensions';
 import { useExtensionBackends } from './useExtensionBackends';
 
+const EXTENSION_DETAILS_URL_BASE =
+  'https://firebase.google.com/products/extensions/';
+
 export function useExtensionsData(): Extension[] {
   const backends = useExtensionBackends();
 
   return convertBackendsToExtensions(backends);
 }
 
-export function convertBackendToExtension(backend: ExtensionBackend) {
+export function convertBackendToExtension(
+  backend: ExtensionBackend
+): Extension {
+  const spec = isLocalExtension(backend)
+    ? backend.extensionSpec
+    : backend.extensionVersion.spec;
+
+  const shared = {
+    authorName: spec.author?.authorName || '',
```

Comment:
In the case that this is a published-extension, we actually want to use the ref's publisher-id, where ref = {publisherId}/{extensionId}